packages/twitch-chat-client/src/Toolkit/UserTools.ts

```diff
@@ -1,7 +1,15 @@
 /** @private */
 export function toUserName(channel: string): string {
 	// it's okay if this is already a user name, we only remove the first character if it's a pound
-	return channel.replace(/^#/, '').toLowerCase();
+	const name = channel.replace(/^#/, '').toLowerCase();
+	// valid usernames must be alphanumberic or underscores between 4 and 25 characters (cannot start with underscore)
+	const re = RegExp(/^[a-zA-Z0-9][a-zA-Z0-9_]{3,24}$/);
```

Comment:
`RegExp()` is an unnecessary wrapper. Also please define this outside the function.